### PR TITLE
Add more apps & Readme description

### PR DESCRIPTION
**Pull request by jan-abel-inwt**
@arunge as we discussed i added more apps & a description of how to add new apps and how to test changes locally.

Let me know if we should remove the under construction banner.

### CODE REVIEW

File: packages.yaml [hunk 1]
```diff
@@ -12,6 +12,43 @@ categories:
       docker_image: "bmscs"
       img: "img/bmscs.jpg"
 
+    - name: "BNR"
```
Comment (arunge) [inline packages.yaml:15]: For BNR the "local install instructions" are not correct. BNR is not a package and the function `startApplication()` does not exist.

image

Correct install instrucions are part of the [README](https://github.com/Pandora-IsoMemo/BNR/blob/main/README.md) currently.

Either we should hide install instructions here or link to the README ...
Also the docker installation looks different here. But it should work also with our docker installation, I think ... :thinking:

**Review by arunge — CHANGES_REQUESTED** [review]
Regarding "reporting bugs":
image

I think we must add a link where to open issues...
Should we add a link to the issues of [docs](https://github.com/Pandora-IsoMemo/docs/issues) (we could shift issues to the respective repository)? Or to the general page of our [organization](https://github.com/Pandora-IsoMemo) (we can ask that the user opens the issue in the respective repository)? What do you think is better?

**Comment by jan-abel-inwt** [comment]
There are still some items missing for BNR and CausalR:

- BNR
  - Documentation
  - Beta Version
- CausalR
  - Documentation
  
 @arunge we should decide if we wanna provide these things for the apps or hide/remove the links.

**Comment by jan-abel-inwt** [comment]
i changed the instructions for apps with rstan and renv. Also i made almost every yaml item optional.

We should adjust the readme in bnr and link to this new installation instruction.

**Comment by jan-abel-inwt** [comment]
> I think we must add a link where to open issues...
Should we add a link to the issues of [docs](https://github.com/Pandora-IsoMemo/docs/issues) (we could shift issues to the respective repository)? Or to the general page of our [organization](https://github.com/Pandora-IsoMemo) (we can ask that the user opens the issue in the respective repository)? What do you think is better?

I think we need to make a distinction here. If it's a bug in a package or app, the issue should be opened in its respective repository. If it's a problem with the installation, we could use this repository instead.

Let's discuss this further tomorrow.

**Pull request merged**